### DISCUSSION TITLE
Stellarium: What stellar object is "Fred"?

### DISCUSSION THREAD

**Discussion by RamoneChook** [Stellarium/stellarium#2372] [Q&A] [answered]
Hello all. After examining Stellarium's custom objects I found something rather unusual named "Fred" in the constellation Draco (with the description: "This object never rises"). I am still very confused on how to describe "Fred".
Can anyone please explain?

**Comment by gzotti** [comment, marked as answer]
"Custom Object" indicates something special. When you are searching for "Fred", the online query to SIMBAD delivers an object which is known to SIMBAD under this name. With this object selected, querying SIMBAD again for "what's there" delivers

At coords (J2000.0) 16:09:18.76 +65:32:50.3:
Object: NAME Fred (Galaxy)
Distance from query:  0.42 arcsec
Other Identifiers:
 - NAME Fred                                          
 - [FMK2006] ACS J160919+6532                         
 - [MWC2015] B1608+656  7926                          
Object Type(s): Galaxy
Spectral Type: ~
Morph. Type: ~
Dimensions:     ~'x    ~' at    ~ degrees

At coords (J2000.0) 16:09:16.3682 +65:32:51.598:
Object: [FPR99] 1608+656 N (Radio-source)
Distance from query: 15.18 arcsec
Other Identifiers:
 - [FPR99] 1608+656 N                                 
Object Type(s): Radio-source,Radio-source
Spectral Type: ~
Morph. Type: ~
Dimensions:     ~'x    ~' at    ~ degrees


The name "Fred" is not something that Stellarium developers have invented. You can search for "Fred" in all available astronomical databases for an explanation what's so special about this object. My guess is that the radio galaxy (2nd entry) emitted some unexplicable signal which was jokingly nicknamed "Fred" by radio astronomers.  But this is my guess, no dependable information.

**Reply by RamoneChook** [reply]
This explains a lot.
Thank you, have a good day. :)
